README.md

```diff
@@ -6,6 +6,38 @@ Create a community of like minded authors to foster inspiration and innovation
 by leveraging the modern web.
 
 ---
+## Getting started 
+These instructions will get you a copy of the project up and running in your local machine for development and testing purposes.
+
+## Prerequisites
+- [Git](https://git-scm.com/download/)
+- [Python 3 and above](https://www.python.org/downloads/)
+- [PostgreSQL](https://www.postgresql.org/)
+
+## Installing
+- Create a working space in your local machine
+- Clone this [repository](https://github.com/andela/ah-django-unchained) `git clone https://github.com/andela/ah-django-unchained.git`
+- Navigate to the project directory
+- Create a virtual environment `python3 -m venv /path/to/new/virtual/environment`
+- Activate your virtual environment `source venv/bin/activate`
+- Install dependencies to your virtual environment `pip install -r requirements.txt`
+
+### Setting up the database
+- Start your database server and create your database
+- Create a .env file and put these key=values in it:
+```
+source venv/bin/activate
```

Comment:
This is assuming that the name of our virtual environment is venv, which is not a right assumption.